phpcs-ruleset.xml.dist

```diff
@@ -28,6 +28,8 @@
         <exclude name="DrupalPractice.InfoFiles.NamespacedDependency.NonNamespaced" />
         <!-- https://github.com/Gizra/og/issues/546 -->
         <exclude name="DrupalPractice.Objects.GlobalClass.GlobalClass" />
+        <!-- https://github.com/Gizra/og/issues/547 -->
+        <exclude name="DrupalPractice.Objects.GlobalDrupal.GlobalDrupal" />
```

Comment:
These two lines are out of scope for this PR. I think they were probably introduced by accident through a merge conflict. Let's remove them.

Reply:
We should delete these lines (they've probably been reintroduced in the merge conflict):
```
        <!-- https://github.com/Gizra/og/issues/546 -->
        <exclude name="DrupalPractice.Objects.GlobalClass.GlobalClass" />

```